src/store/network/hooks.ts

```diff
@@ -3,10 +3,10 @@
  *
  * SPDX-License-Identifier: AGPL-3.0-only
  */
-import { useNetworkStore } from './store';
+import { useInfoRefresh, useSync } from '@zextras/carbonio-ui-soap-lib';
+
 import type { SoapNotify, SoapRefresh } from '../../types/network';
 
-const FALLBACK_NOTIFY: SoapNotify[] = [];
-const FALLBACK_REFRESH: SoapRefresh = {};
-export const useNotify = (): SoapNotify[] => useNetworkStore((s) => s.notify ?? FALLBACK_NOTIFY);
-export const useRefresh = (): SoapRefresh => useNetworkStore((s) => s.refresh ?? FALLBACK_REFRESH);
+export const useNotify = (): SoapNotify[] => useSync();
+
+export const useRefresh = (): SoapRefresh => useInfoRefresh();
```

Comment:
https://github.com/search?q=org%3Azextras+useRefresh%28%29&type=code
there are 2 usage: can we remove this hook and directly replace with the new one? or at least we need to mark as deprecated and suggest the new available hook